html/browsers/history/the-location-interface/location-ancestor-origins-referrerpolicy-snapshot.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<title>location.ancestorOrigins snapshot timing of referrerpolicy</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+promise_test(async () => {
+  assert_implements(location.ancestorOrigins);
+  const iframe = document.createElement('iframe');
+  iframe.src = '/common/blank.html?pipe=trickle(d1)';
+  iframe.referrerPolicy = 'no-referrer';
+  const loaded = new Promise(resolve => {
+    iframe.onload = () => {
+      resolve();
+    };
+  });
+  document.body.append(iframe);
+  // The referrerpolicy attribute was snapshotted when the initial about:blank doc was created
+  // https://html.spec.whatwg.org/#the-iframe-element:create-a-new-child-navigable
+  // https://html.spec.whatwg.org/#child-navigables:creating-a-new-browsing-context
+  assert_array_equals(Array.from(iframe.contentWindow.location.ancestorOrigins), ['null']);
+  iframe.referrerPolicy = '';
```

Comment:
Maybe let's leave a comment here saying that this should have no effect, because the referrer policy used for ancestor origin redaction (and embedder-initiated iframe src fetching) is snapshotted at navigation time, since that's the direction the PR went.